### ISSUE TITLE
Tab-based navigation in timepicker with enabled spinners

### ISSUE THREAD

**Issue by tilowestermann** [ng-bootstrap/ng-bootstrap#1836]
### Bug description:

Using a timepicker with spinners enabled, I expect that I can change from the hours field to the minutes field using tab on the keyboard. While this works without spinners, it doesn't work with spinners enabled. Here, pressing tab selects the first spinner control.

### Link to minimally-working plunker that reproduces the issue:

See demo page: https://ng-bootstrap.github.io/#/components/timepicker/examples

Example showing the requested behaviour: http://angular-ui.github.io/bootstrap/#!#%2Ftimepicker

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: 4.3.6

ng-bootstrap: 1.0.0-beta.2

Bootstrap: 4.0.0-beta

**Comment by MeMeMax** [comment]
Is this planned to be fixed?

**Comment by MeMeMax** [comment]
@pkozlowski-opensource I find this issue as closed but it isn´t fixed. Is that correct?
